requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 altair
 cartopy
+descartes
```

Comment:
I couldn't find `descartes`, do you know where it was used?

Reply:
tl;dr it;s on geopandas not handling dependencies properly, but essentially it breaks 
`TestNotebooks.test_time_slider_choropleth`

<details>
<summary>Full traceback</summary>

~~~

_______________________________________ TestNotebooks.test_time_slider_choropleth _______________________________________

self = <test_notebooks.NotebookTester object at 0x7f755a854e48>, exporter = None, filename = None

    def __call__(self, exporter=None, filename=None):
        raw_nb = nbconvert.exporters.Exporter().from_filename(self.filename)
        raw_nb[0].metadata.setdefault('kernelspec', {})['name'] = 'python'
>       exec_nb = nbconvert.preprocessors.ExecutePreprocessor(timeout=600).preprocess(*raw_nb)

tests/notebooks/test_notebooks.py:29: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
venv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py:262: in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
venv/lib/python3.5/site-packages/nbconvert/preprocessors/base.py:69: in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <nbconvert.preprocessors.execute.ExecutePreprocessor object at 0x7f7556331eb8>
cell = {'execution_count': 3, 'outputs': [{'ename': 'ImportError', 'output_type': 'error', 'evalue': 'The descartes package i...d': 'light'}}], 'source': '%matplotlib inline\n\nax = gdf.plot(figsize=(10, 10))', 'cell_type': 'code', 'metadata': {}}
resources = defaultdict(None, {'output_extension': '.txt', 'metadata': defaultdict(None, {'name': 'TimeSliderChoropleth', 'path': '/home/demetrio/Parkbob/git/folium/tests/notebooks/../../examples', 'modified_date': 'September 3, 2018'})})
cell_index = 3

    def preprocess_cell(self, cell, resources, cell_index):
        """
            Executes a single code cell. See base.py for details.
    
            To execute all cells see :meth:`preprocess`.
            """
        if cell.cell_type != 'code':
            return cell, resources
    
        reply, outputs = self.run_cell(cell, cell_index)
        cell.outputs = outputs
    
        if not self.allow_errors:
            for out in outputs:
                if out.output_type == 'error':
>                   raise CellExecutionError.from_cell_and_msg(cell, out)
E                   nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
E                   ------------------
E                   %matplotlib inline
E                   
E                   ax = gdf.plot(figsize=(10, 10))
E                   ------------------
E                   
E                   ---------------------------------------------------------------------------
E                   ImportError                               Traceback (most recent call last)
E                   ~/Parkbob/git/folium/venv/lib/python3.5/site-packages/geopandas/plotting.py in plot_polygon_collection(ax, geoms, values, color, cmap, vmin, vmax, **kwargs)
E                        81     try:
E                   ---> 82         from descartes.patch import PolygonPatch
E                        83     except ImportError:
E                   
E                   ImportError: No module named 'descartes'
E                   
E                   During handling of the above exception, another exception occurred:
E                   
E                   ImportError                               Traceback (most recent call last)
E                   <ipython-input-3-423c72b95e2e> in <module>()
E                         1 get_ipython().run_line_magic('matplotlib', 'inline')
E                         2 
E                   ----> 3 ax = gdf.plot(figsize=(10, 10))
E                   
E                   ~/Parkbob/git/folium/venv/lib/python3.5/site-packages/geopandas/geodataframe.py in plot(self, *args, **kwargs)
E                       532         from there.
E                       533         """
E                   --> 534         return plot_dataframe(self, *args, **kwargs)
E                       535 
E                       536     plot.__doc__ = plot_dataframe.__doc__
E                   
E                   ~/Parkbob/git/folium/venv/lib/python3.5/site-packages/geopandas/plotting.py in plot_dataframe(df, column, cmap, color, ax, categorical, legend, scheme, k, vmin, vmax, markersize, figsize, legend_kwds, **style_kwds)
E                       412         return plot_series(df.geometry, cmap=cmap, color=color, ax=ax,
E                       413                            figsize=figsize, markersize=markersize,
E                   --> 414                            **style_kwds)
E                       415 
E                       416     # To accept pd.Series and np.arrays as column
E                   
E                   ~/Parkbob/git/folium/venv/lib/python3.5/site-packages/geopandas/plotting.py in plot_series(s, cmap, color, ax, figsize, **style_kwds)
E                       292         values_ = values[poly_idx] if cmap else None
E                       293         plot_polygon_collection(ax, polys, values_, facecolor=facecolor,
E                   --> 294                                 cmap=cmap, **style_kwds)
E                       295 
E                       296     # plot all LineStrings and MultiLineString components in same collection
E                   
E                   ~/Parkbob/git/folium/venv/lib/python3.5/site-packages/geopandas/plotting.py in plot_polygon_collection(ax, geoms, values, color, cmap, vmin, vmax, **kwargs)
E                        82         from descartes.patch import PolygonPatch
E                        83     except ImportError:
E                   ---> 84         raise ImportError("The descartes package is required"
E                        85                           " for plotting polygons in geopandas.")
E                        86     from matplotlib.collections import PatchCollection
E                   
E                   ImportError: The descartes package is required for plotting polygons in geopandas.
E                   ImportError: The descartes package is required for plotting polygons in geopandas.

venv/lib/python3.5/site-packages/nbconvert/preprocessors/execute.py:286: CellExecutionError
~~~

</details>